db/migrate/20200910145925_remove_vim_types_from_customization_specs.rb

```diff
@@ -0,0 +1,40 @@
+class RemoveVimTypesFromCustomizationSpecs < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+  include MigrationHelper
+
+  class CustomizationSpec < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  def up
+    say_with_time("Removing Vim Types from CustomizationSpecs") do
+      base_relation = CustomizationSpec.in_my_region.where("spec LIKE ?", "%hash-with-ivars:VimHash%")
```

Comment:
note: this ensures records already processed won't get processed again

Reply:
I notice that the regular expression in the `update_all` matches more cases than this like in the limiter / `base_relation`.

Is it possible to only have something like a `string:VimString` and it will not trigger the base relation but should be updated by the `update_all`?